### ISSUE TITLE
Working with genomes with more than 23 chromosomes

### ISSUE THREAD

**Issue by tottlefields** [frankvogt/vcf2gwas#20]
Firstly, great work, thank you!

I work with dog data and as such have 38 chromosomes in my VCF file. Your manual states "_vcf2gwas recognizes chromosomes in the following formats (here the first chromosome): Chr1, chr1, 1._", which it does. However, when using chr37 (in this case) I get an error as an extra PLINK flag is required... I have tried with chr37 and 37 as I can see a numerical check happening in the **make_bed** function.

```
Converting to PLINK BED..
Error: Invalid chromosome code '37' on line 2267 of .vcf file.
(This is disallowed for humans.  Check if the problem is with your data, or if
you forgot to define a different chromosome set with e.g. --chr-set.)
```

How can I run this analysis on genomes with more than 23 chromosomes please?
Thanks :)

**Comment by frankvogt** [comment]
Hi,

Is it possible that in your VCF file some chromosomes are missing?
The issue might be that vcf2gwas checks the total number of chromosomes and sets the plink flags accordingly while plink may still have a problem if a chromosome is labeled with "37" even though the total number of chromosomes in your VCF file would be less than 37.

**Comment by tottlefields** [comment]
Ahhh, OK. Yes I was running it on a single chromosome VCF file as that is how we store our VCF files (else they are too big to manage and it’s easier to parallelise across individual car files). Your program ran absolutely fine on a single chr24 so assume dit was a human versus dog issue (we hit these fairly regularly, LOL!). I managed to get it to run labelling the car as 3.7 rather than 37 so I have a work around, but it would be fab if it could work for chromosomes larger than 24 the asme way that it works with less than 24.

Thanks,
Ellen.

**Comment by frankvogt** [comment]
Alright perfect! I will implement a fix for this case as soon as possible.

**Comment by frankvogt** [comment]
fixed in vcf2gwas v0.8.9
